app/services/cfe/regular_transactions_payload_service.rb

```diff
@@ -5,13 +5,16 @@ def call
         relevant_form?(:other_income) ||
         relevant_form?(:mortgage_or_loan_payment) ||
         relevant_form?(:housing_costs) ||
+        relevant_form?(:shared_ownership_housing_costs) ||
```

Comment:
I wonder whether we should be making the changes to the CFE payload in a separate ticket/pull request?

Reply:
it could but it makes sense to go int his ticket really as the new form is created in this ticket